frontend/src/api/mockAPI/contract_templates.js

```diff
@@ -39,9 +39,18 @@ export class ContractTemplate extends MockAPIController {
             if (message) {
                 throw new Error(message);
             }
+            return;
         }
     }
     upload({ file_contents, file_name }) {
+        const file_names = Object.keys(
+            this.data.contract_templates_by_filename
+        );
+        if (file_names.includes(file_name)) {
```

Comment:
You can just do `if (file_name in this.data.contract_templates_by_filename)`

Reply:
Or maybe more clear `if(this.data.contract_templates_by_filename[file_name] != null)`